### ISSUE TITLE
Failed to Compile with Laravel Spark

### ISSUE THREAD

**Issue by randyrankin** [spatie/laravel-mix-purgecss#20]
First off, thank you and great work! I hit a bit of a problem with a Laravel Spark 6 project. `npm run prod` produces the following::

```
 ERROR  Failed to compile with 50 errors                                                                         7:35:23 AM

These dependencies were not found:
auth/register-braintree in ./resources/assets/js/spark-components/bootstrap.js
auth/register-stripe in ./resources/assets/js/spark-components/auth/register-stripe.js
kiosk/add-discount in ./resources/assets/js/spark-components/kiosk/add-discount.js
kiosk/announcements in ./resources/assets/js/spark-components/kiosk/announcements.js
...
```

**Comment by sebastiandedeyne** [comment]
This doesn't really look like it has anything to do with this package...

Are you sure you're not encountering this issue without `.purgeCss()`?

If it's still a problem, please share the contents of your `package.json` and the output of `npm ls`

**Comment by randyrankin** [comment]
I found the problem: I removed `node_modules` folder and ran `yarn`, which returned `laravel-mix-purgecss@1.0.5 requires a peer of laravel-mix@^2.0.0 but none is installed.`. My `package.json` file was referring to `"laravel-mix": "1.*"`.  I changed to `"laravel-mix": "2.*"`, ran `yarn` again, then `npm run prod` (with `purgeCss()` in `webpack.mix.js`) and all ran fine. 

Thank you for responding so quickly, especially on a Saturday. I really appreciate all the hard work that you and the Spatie team put into these packages. Many thanks.
